dynamodb/src/test/scala/zio/dynamodb/codec/SetCodecSpec.scala

```diff
@@ -0,0 +1,140 @@
+package zio.dynamodb.codec
+
+import zio.dynamodb.{ AttributeValue, Codec }
+import zio.schema.{ DeriveSchema, Schema }
+import zio.test.Assertion.{ equalTo, isRight }
+import zio.test._
+
+import java.math.BigInteger
+import scala.collection.immutable.Map
+
+object SetCodecSpec extends DefaultRunnableSpec {
+
+  final case class HasBigDecimalSet(set: Set[BigDecimal])
+  object HasBigDecimalSet        {
+    implicit val schema: Schema[HasBigDecimalSet] = DeriveSchema.gen[HasBigDecimalSet]
+  }
+  final case class HasJavaBigDecimalSet(set: Set[java.math.BigDecimal])
+  object HasJavaBigDecimalSet    {
+    implicit val schema: Schema[HasJavaBigDecimalSet] = DeriveSchema.gen[HasJavaBigDecimalSet]
+  }
+  final case class HasBigIntSet(set: Set[BigInt])
+  object HasBigIntSet            {
+    implicit val schema: Schema[HasBigIntSet] = DeriveSchema.gen[HasBigIntSet]
+  }
+  final case class HasJavaBigIntegerSet(set: Set[BigInt])
```

Comment:
It looks like this and `HasBigIntSet` are the same?

Reply:
:+1: